CITi-UFPE/hackapride_2022
# Feature/mobile contact

# What I did?

- Mobile screen

## How to test?

1. Open the terminal and go to project folder

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/mobile-contact`
   1. `git pull origin feature/mobile-contact`
   1. `yarn install` ou `npm install`
   1. `code .`
   1. `yarn start` ou `npm start`

1. In browser go to http://localhost:3000/

1. Verify if the component or section looks like the mockup with diferents sizes of screens

1. Read section change files on GitHub

1. Test the things if you think that is necessary test, even those that are not in this description
